src/components/CardList.spec.js

```diff
@@ -0,0 +1,10 @@
+import { render, screen } from '@testing-library/react';
+import CardList from './CardList';
+
+describe('CardList', () => {
+  it('Render a list of vocabulary cards', () => {
```

Comment:
You can write here `it('renders a list of vocabulary cards',()...` 🙂  like in your `Card.spec.js`